File: src/_1ProgrammingBasicsWithJavaMay2023/_5WhileLoop/_1Lab/_07MinNumber.java
package _1ProgrammingBasicsWithJavaMay2023._5WhileLoop._1Lab;

import java.util.Scanner;

public class _07MinNumber {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        String input = scanner.nextLine();
        int minNumber = Integer.MAX_VALUE;
        while (!input.equals("Stop")) {
            int number = Integer.parseInt(input);
            if (number < minNumber) {
                minNumber = number;
            }
            input = scanner.nextLine();
        }
        System.out.printf("%d",minNumber);
    }
}

File: src/_1ProgrammingBasicsWithJavaScriptNovember2024/_5WhileLoop/_1Lab/_07MinNumber.js
function minNumber(input) {
    let command = input[0];
    let smallestNumber = Number.MAX_SAFE_INTEGER;

    let counter = 1;
    while (command !== "Stop") {
        const number = Number(command);

        if (number < smallestNumber) {
            smallestNumber = number;
        }

        command = input[counter++];
    }

    console.log(`${smallestNumber}`);
}